docs/resources/deployment.md

```diff
@@ -100,9 +105,10 @@ Required:
 
 Optional:
 
-- `git_sha1` (String) The git object hash for the file. This is valid only for kind == "file".
-- `target` (String) The target file path for the symlink. This is valid only for kind == "symlink".
-- `updated_at` (String) The time the file was last updated. This is valid only for kind == "file".
+- `content` (String) The inlined content of the asset. This is valid only for `file` asset. If both `content` and `local_file_path` are specified, `content` will have higher precedence.
```

Comment:
Rather than specifying that one takes precendence over the other, why not make them mutually exclusive?

Reply:
We could do that. I'll fix that

Reply:
Fixed and added tests [fb7d120](https://github.com/denoland/terraform-provider-deno/pull/28/commits/fb7d12081e9c41334704930133618060e7d93bca)